frontend/src/components/TodoForm.tsx

```diff
@@ -1,8 +1,9 @@
 import {Todo} from "../model";
-import {useState} from "react";
-import "./TodoForm.css"
+import {useEffect, useState} from "react";
 import {useTranslation} from "react-i18next";
 import LanguageSelection from "./LanguageSelection";
+import {Link} from "react-router-dom";
+import About from "./About";
```

Comment:
Das wird in der Datei nicht verwendet und solltet gelöscht werden.

Reply:
Was sollte gelöscht werden? Meinst du den Link in Zeile 53?

Reply:
ok erledigt